libraries/adsk/adsklib/colorcorrect.mtlx

```diff
@@ -3,8 +3,8 @@
 
    <!-- Sample color correct interface. Versioned and namespaced -->
    <nodedef name="ND_colorcorrect" node="colorcorrect" nodegroup="adjustment" version="1.0" isdefaultversion="true" namespace="adsk">
-      <parameter name="color" type="color3" value="0.5, 0.5, 0.5" />
-      <parameter name="alpha" type="float" value="1" />
+      <input name="color" type="color3" value="0.5, 0.5, 0.5" />
```

Comment:
This failed for code gen as we don't allow upstream traversal from a parameter when we create an instance of this, making this node pretty well useless for anything other than modifying the constants in the definition.